examples/menu_example.py

```diff
@@ -45,7 +45,7 @@ def __init__(self):
 
         self.set_default_size(200, 200)
 
-        action_group = Gtk.ActionGroup("my_actions")
+        action_group = Gtk.ActionGroup.new("my_actions")
```

Comment:
Use keyword arguments instead of `.new`.

Reply:
Can we retain for `DeprecationWarning` as with `GtkTable`